assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -84,8 +111,39 @@ const WPDashboardWidgets = () => {
 					) }
 				</div>
 			) }
+
 			{ analyticsModuleActive && analyticsModuleConnected && (
-				<WPDashboardPopularPagesWidget />
+				<Fragment>
+					<WPDashboardUniqueVisitorsWidget />
+					<div className={ classnames(
+						'googlesitekit-wp-dashboard-stats__area-wrapper',
+						{
+							[ HIDDEN_CLASS ]: shouldCombineAnalyticsArea1,
+						} ) }>
```

Comment:
Let's try to solve this without adding extra wrapper elements which may disrupt the layout in unexpected ways and increase the complexity of the CSS (see the extra CSS you needed to add to make this work).

It might be more appropriate to add a new prop to the individual widgets themselves, e.g. a `className` where you could pass through the `HIDDEN_CLASS`, so that the class is added on existing DOM elements rather than new wrapper elements.

Reply:
I was a bit daunted by the prospect of prop-drilling `className` down through the various widgets to their CTAs, but now I look at it again, you are right, I think this can be done simply. Thanks 👍

Reply:
@johnPhillips Actually, I see what you're saying. The updated approach has its own problems since you need to update low-level components like `ReportZero` just for this "special case". Also it now results in some inconsistent behavior since passing a `className` to one of the widget components only affects the `ReportZero` version of that widget, which would be unexpected.

I think we need to do it yet another way:
* Instead of altering the classes of the widgets in place, let's render them in an entirely different location if they need to be hidden.
* Specifically, if any of the widgets need to be hidden, let's render a `div` container with `HIDDEN_CLASS` somewhere e.g. at the end of the overall dashboard widget and put all the widgets to hide in there, unmodified.
* This ensures the widgets still get "rendered", but without potentially disrupting the main parts of the DOM, and without making all those changes to pass through class names.

Sorry on the change of course here again, but your update made it clear another approach is needed. The one above is actually closest to how regular widgets behave.

Reply:
No problem @felixarntz 
Agreed, this feels like the more solid option. I've implemented your suggestion - let me know what you think 👍